src/styles/header.scss

```
@@ -0,0 +1,69 @@
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin: 0 auto;
+  padding-left: 50px;
+  padding-right: 50px;
+  height: 60px;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+}
+
+.img {
+  display: flex;
+}
+
+.navigation__list {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  list-style: none;
+  height: 60px;
+}
+
+.navigation__item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.navigation__link {
+  position: relative;
+  display: flex;
+  align-items: center;
+  height: 60px;
+  color: $text-color-dark;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 14px;
+  text-transform: uppercase;
+  text-decoration: none;
+  transition-duration: 300ms;
+
+  &:hover {
+    color: $nav-active-color;
+    transition-duration: 300ms;
```

move it to the `link` like a
```suggestion
    transition: color $transitionDuration;
```